e2e/tests/node-schema.e2e-spec.ts

```diff
@@ -0,0 +1,150 @@
+import { AppPage } from '../page-objects/app.po';
+import { NodeAdmin } from '../page-objects/node-admin.po';
+import { NodeSchema } from '../page-objects/node-schema.po';
+
+describe('node schema', () => {
+    let admin: NodeAdmin;
+    let page: AppPage;
+    let schema: NodeSchema;
+
+    beforeEach(async () => {
+        admin = new NodeAdmin();
+        page = new AppPage();
+        schema = new NodeSchema();
+        await page.navigateToHome();
+        await admin.modeChange();
+        await schema.openSchema();
+    });
+
+    it('should equal schema names', async () => {
+        const list: Array<String> = ['category', 'vehicleImage', 'vehicle', 'binary_content', 'folder'];
+
+        await expect(schema.getAllSchemasName()).toEqual(list);
```

Comment:
Same as microschema (create schemas by api, etc.)

Reply:
Functions for create/delete a microschema were created in api.ts. BeforeAll method to create test microschemas and afterAll method to delete test microschemas were implemented.